common4j/src/main/com/microsoft/identity/common/java/controllers/CommandResult.java

```diff
@@ -31,40 +31,79 @@
 import java.util.Map;
 
 import lombok.Getter;
+import lombok.NonNull;
 import lombok.Setter;
 import lombok.experimental.Accessors;
 
-public class CommandResult implements ICommandResult {
-
-    public String getCorrelationId() {
-        return mCorrelationId;
-    }
+public class CommandResult<T> implements ICommandResult<T> {
 
+    @Getter
+    @Accessors(prefix = "m")
     private final ResultStatus mStatus;
-    private final Object mResult;
+
+    @Getter
+    @Accessors(prefix = "m")
+    private final T mResult;
+
+    @Getter
```

Comment:
you can add getter and accessor on top the class once

Reply:
Yup.